starknet_devnet/fee_token.py

```diff
@@ -83,28 +80,20 @@ async def get_balance(cls, address: int) -> int:
         return balance
 
     @classmethod
-    async def mint_lite(cls, to_address: int, amount: int) -> None:
-        """Mint `amount` of token at `to_address` without creating a tx."""
-        assert cls.contract
-        amount_uint256 = Uint256.from_felt(amount)
-        await cls.contract.mint(to_address, (amount_uint256.low, amount_uint256.high)).invoke()
-
-    @classmethod
-    async def mint(cls, to_address: int, amount: int, starknet_wrapper):
-        """Mint `amount` of token at `to_address` with creating a tx."""
-        assert cls.contract
-        amount_uint256 = Uint256.from_felt(amount)
-
+    def get_mint_transaction(cls, to_address: int, amount: Uint256):
+        """Construct a transaction object representing minting request"""
         transaction_data = {
             "entry_point_selector": hex(get_selector_from_name("mint")),
             "calldata": [
                 str(to_address),
-                str(amount_uint256.low),
-                str(amount_uint256.high),
+                str(amount.low),
+                str(amount.high),
             ],
             "signature": [],
             "contract_address": hex(cls.ADDRESS)
         }
-        transaction = InvokeFunction.load(transaction_data)
-        _, tx_hash, _ = await starknet_wrapper.invoke(transaction)
-        return tx_hash
+        return InvokeFunction.load(transaction_data)
+
+    async def mint_lite(self, to_address: int, amount: Uint256) -> None:
```

Comment:
Can we add mint_lite also to starknet_wrapper so that similar functions are nearby?

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/202/commits/8691e6aae1d3d329e28b8fc7917d25944f7246fb